client/src/components/charts/LineChart.jsx

```diff
@@ -365,7 +364,7 @@ export default class LineChart extends Component {
 
                   <AxisLeft
                     scale={yScale}
-                    left={xScale(series.data[0].timestamp)}
+                    left={xScale(0)}
```

Comment:
this means the axis will be inline with the value 0 instead of the first node

Reply:
and that value is always `0` ?

Reply:
yes the ticket says it should always be `0`